tasks/checks.yml

```diff
@@ -14,3 +14,10 @@
   set_fact:
     concourseci_download_url: "{{ concourseci_download_fr_url }}"
   when: "{{ 'rc' not in concourseci_version }}"
+
+- name: check | Determine if systemctl is installed
```

Comment:
this task not needed if you use ```ansible_service_mgr == "systemd"``` avoid giving a false error even if it's ignored